app/models/network/builders/heat.rb

```diff
@@ -16,23 +18,26 @@ def to_graph
 
       private
 
-      def initialize(tree, techs, heat_sources)
-        @tree         = tree
-        @techs        = techs
+      def initialize(tree, techs, heat_sources, opts)
+        @tree = tree
+        @techs = techs
         @heat_sources = heat_sources
+        @capacity_per_conn = opts[:central_heat_buffer_capacity] || 10.0
       end
 
       def build_graph
         # TODO Create production park on installed techs.
         dispatchable, must_run =
           sources_to_producers(@heat_sources).partition(&:dispatchable?)
 
+        base_volume = number_of_connections * 4 * @capacity_per_conn
+
         @park = Network::Heat::ProductionPark.new(
           must_run:         must_run,
           dispatchable:     dispatchable,
           # https://github.com/quintel/etmoses/issues/971
-          volume:           number_of_connections * 4 * 10.0,
-          amplified_volume: number_of_connections * 4 * 17.78
+          volume:           base_volume,
+          amplified_volume: base_volume * 1.78
```

Comment:
I've expanded on the comment with information about the number; I agree that the comment should have just described it in the first place, instead of directing the reader to a GitHub issue. :+1: